.github/workflows/ci.feature.yml

```diff
@@ -155,6 +155,12 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
+      
```

Comment:
what was the reason for this change? anything worth documenting here in case it's accidentally reverted?

Reply:
a lot of "could not connect to network errors" which occurred somewhat randomly, I believed this was due to a race condition where between the `ganache-cli --networkId 4447 -h 0.0.0.0 -m "test test test test test test test test test test test junk" &` and `yarn test` commands.

Reply:
okay, fair enough. Usually in testing scripts, a sleep 5 is added, but here it follows a long-period command which gives the same effects.